src/utils/index.ts

```diff
@@ -0,0 +1,9 @@
+export function addZero(department: number): string {
+    const shouldRemoveZero = department < 10 && !String(department).includes('0');
```

Comment:
Department is a number (it's parsed as a number at the end of the file, the .map(Number)), so it can't possibly be >0 and <10 and have zero